source/_integrations/portainer.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Portainer
+description: Instructions on how to integrate Portainer with Home Assistant.
+ha_category:
+  - Binary sensor
+ha_release: 2025.7
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@erwindouna'
+ha_domain: portainer
+ha_config_flow: true
+ha_platforms:
+  - binary_sensor
+ha_integration_type: integration
+---
+
+The Portainer {% term integration %} is used as an interface to the [Portainer API](https://docs.portainer.io/api).
+Portainer is a lightweight management UI that allows you to easily manage your Docker containers, images, networks, and volumes. It works on every Docker host or Swarm cluster.
+
+The Portainer API provides a way to manage Docker containers, images, networks, and volumes. It allows you to interact programmatically with your Docker host or Swarm cluster.
+
+## Prerequisites
+
+Before you can configure Portainer within Home Assistant, you need an Access Token. Create a Portainer Access Token by following these steps:
```

Comment:
Are there any other components needed before I can add the integration?
Like a Portainer account?